csharp-basics/exercises/Collections/Exercise3/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Transactions;
+
+namespace Exercise3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter name count in list :   ");
+            int nameCount = int.Parse(Console.ReadLine());
```

Comment:
Un kāpēc ir vajadzīgs šis?

Reply:
Neuzmanīgi izlasiju uzdevumu nosacijumus.